source/integrate/community-for-mattermost-for-microsoft-teams.rst

```diff
@@ -0,0 +1,71 @@
+Community for Mattermost for Microsoft 365
+==========================================
+
+.. include:: ../_static/badges/ent-cloud-selfhosted.rst
```

Comment:
This isn't really "available" in the same way our other offerings are -- in fact, you could be a non-Mattermost customer and use this. Should I just rip out these badges?

Reply:
That's my recommendation, yes.

Reply:
Removed plan/deploy badges